### ISSUE TITLE
Ability to Duplicate Group

### ISSUE THREAD

**Issue by Matt-Power** [Bruker-Amics/bugs_features#51]
# Problem
Often I need similar grouping (for instance, one with the feldspars split out, one with them all grouped but all other minerals are the same) - reconstructing the list is very time consuming. It is possible to import from another measurement but this workaround is unwieldy.

# Solution 
Add a duplicate group button on the group tab of the classification 

# Acceptance Criteria 
- [ ] the user can duplicate a grouping on the classification grouping tab

**Comment by AHM-amics** [comment]
I agree with Matt and this would be very useful. In other words, create a duplicate mineral list to a previous one so that it can be edited and renamed.

**Comment by jonathan-knapp** [comment]
@Matt-Power am I correct you want to duplicate the WORKFLOW and not the GROUP inside the workflow?

**Comment by Matt-Power** [comment]
No, I simply want to be able to duplicate a group within the current sample (e.g. group 1 and then group 1 (Copy)) without using the Workflow tab. Use case is that I have a detailed grouping with, say, loads of different Cu minerals but I also want a simplified group for simplified reporting or extracting a specific data type. I would duplicate the detailed grouping and then quickly merge all of the Cu minerals together. 

Currently if I want to duplicate the group, I have to import from an existing measurement (which is clunky and involves several button clicks). It also means that if I have several groups already, all will be duplicated and I have to then delete the ones I don't want - which is really clunky.

Either a duplicate button or an additional option under the current Add Group button would work.

**Comment by jonathan-knapp** [comment]
@AlexeiDolgolyov - this one is #51 - its kind of embarrassing it's stuck around this long.!

**Comment by AlexeiDolgolyov** [comment]
<img width="642" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlV3Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c0fbf5193a308f478919ccf797e2c20375a6babf/image.png" alt="image.png" />

<img width="647" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNll3Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2a998ba8690b36a5bbc76f614e2e4f484b4178c5/image.png" alt="image.png" />
